repos.bzl

```diff
@@ -2,944 +2,58 @@ load("@bazel_gazelle//:deps.bzl", "go_repository")
 
 def go_repositories():
```

Comment:
Bit of a large diff here, though looks like a lot of reorderings - this was generated via `gazelle update-repos -from_file=go.mod -to_macro=repos.bzl%go_repositories` from the authentikos directory. Let me know if there's a better way to update this file!